recipe/meta.yaml

```
@@ -55,6 +55,7 @@ test:
     - pytest-cov >=2.2.0
     - ipython >=2.0
     - boto3 >=1.2.4
+    - fsspec
     - matplotlib-base
     - packaging
     - hypothesis
```

Need `aiohttp` and `requests` too I think to fix `ImportError: HTTPFileSystem requires "requests" and "aiohttp" to be installed`

```suggestion
    - aiohttp
    - boto3 >=1.2.4
    - fsspec
    - matplotlib-base
    - packaging
    - requests
    - hypothesis
```